Prox/Prox/Utilities/Logger.swift

```diff
@@ -0,0 +1,66 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+
+enum LoggerLevel: Int {
+    case debug
+    case info
+    case warn
+    case error
+}
+
+class Logger {
+    private let tag: String
+
+    /// All log levels >= this level will be printed.
+    var level: LoggerLevel
```

Comment:
nit: maybe -> `minPrintedLevel` or `maxPrintedLevel` and do away with the comment (which is more likely to go out of date)?